frontend/src/Components/MainContainer/MainContainer.module.css

```diff
@@ -89,6 +91,43 @@
   margin-top: 23px;
 }
 
+@media screen and (max-width: 1280px) and (max-height: 720px) {
+  .paragraph {
+    font-size: 28px;
+  }
+
+  .title {
+    font-size: 60px;
+  }
+
+  .mainContainer {
+    
+    height: 100vh;
+  }
+
+  .questionsContainer {
+    padding: 0%;
```

Comment:
Why use percentages for padding?

Reply:
In this case I wanted to remove the padding and I just clicked on the percentages option

Reply:
You can leave it at 0 only then.